### PR TITLE
PR | Issue-33 | Main layout for email page implemented

### PR DESCRIPTION
**Pull request by VladislavGulik**

### CODE REVIEW

File: build/src/email.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Email</title>
+</head>
+<body>
+    <main class="main main_email-page">
+        <div class="main__container">
+            <div class="main main_email-filter-bar">
+                <div class="items items_left">
```
Comment (arsenykonohovwork) [inline build/src/email.html]: The save as for #39;

File: build/src/email.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                </div>
+                <div class="items items_right">
+                    <span></span>
+                    <select name="" id="">
+                        <option value="date">Date</option>
+                    </select>
+                </div>
+            </div>
+            <div class="list vertical list_conversation">
+                <ul>
```
Comment (arsenykonohovwork) [inline build/src/email.html]: Where is the full elements composition for list block?

File: build/src/email.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            </div>
+            <div class="main chat-thread">
+                <img src="" alt="chat-photo">
+                <span class="main message"></span>
+                <span class="main message-info"></span>                
+                <input type="text" value="Wrire a message">
+            </div>
+            <div class="main profile-info">
+                <img src="" alt="photo">
+                <span class="main__name-surname"></span>
```
Comment (arsenykonohovwork) [inline build/src/email.html]: this features below just text. It's not `.main` block elements.

**Comment by VladislavGulik** [comment]
I fix the code for this PR. Check out last commit and approve if everything is fine, please.

File: build/src/index.html [hunk 4]
```diff
@@ -5,16 +5,91 @@
...
+                        <select name="" id="">
+                            <option value="date">Date</option>
+                        </select>
+                    </div>
+                </div>
+            </div>
+            <div class="main_body">
+                <div class="list vertical list_conversation">
+                    <ul class="list__list_vertical">
+                        <li class="list__item">
```
Comment (arsenykonohovwork) [inline build/src/index.html]: @VladislavGulik  Where is the `item` starts?

File: build/src/index.html [hunk 5]
```diff
@@ -5,16 +5,91 @@
...
+                    </ul>
+                    <button>New conversation</button>
+                </div>
+                <div class="main chat-thread">
+                    <img src="" alt="chat-photo">
+                    <span class="message"></span>
+                    <span class="message-info"></span>                
+                    <input type="text" value="Write a message">
+                </div>
+                <div class="main profile-info">
```
Comment (arsenykonohovwork) [inline build/src/index.html]: It seems to me you have to modify this section

File: build/src/index(main).html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.main.css">
+    <title>Dark-Mockup-UI</title>
+</head>
+<body>
+    <main class="main-section_home-page">
+        <div class="main-section_home-page__container">
```
Comment (arsenykonohovwork) [inline build/src/index(main).html]: It seems to me best to do this one like:
```
.main
   section.main__container.section.section_mod1
   section.main__container.section.section_mod1
   ....
```

**Comment by VladislavGulik** [comment]
@arsenykonohovwork, i resolve problems, and branch is ready for merge.

**Comment by VladislavGulik** [comment]
@arsenykonohovwork, i fixed main layout, please check it and let me now if something wrong.

**Pull request merged**